CHANGELOG.md

```diff
@@ -404,6 +404,7 @@ BREAKING CHANGES:
   * `client.enabled` now defaults to `false`. Setting it to `true` will deploy client agents, however, none of the consul-k8s components will use clients for their operation.
   * `global.imageEnvoy` is no longer used for sidecar proxies, as well as mesh, terminating, and ingress gateways.
   * `externalServers.grpcPort` default is now `8502` instead of `8503`.
+  * `externalServers.hosts` no longer supports [cloud auto-join](https://developer.hashicorp.com/consul/docs/install/cloud-auto-join) strings directly. Instead, include an [`exec=`](https://github.com/hashicorp/go-netaddrs#command-line-tool-usage) string in the `externalServers.hosts` list to invoke the `discover` CLI. For example, the following string invokes the `discover` CLI with a cloud auto-join string: `exec=discover -q addrs provider=aws region=us-west-2 tag_key=consul-server tag_value=true`. The `discover` CLI is included in the official `hashicorp/consul-dataplane` images by default.
```

Comment:
I assume I need to update the Github release as well.

Do I need to add something to the .changelog directory (or should I)?

Reply:
Yeah you'll need to update the Github release

Just thinking about this... If you add it to the .changelog directory, I think it will pick it up the next time we try to generate the changelog for the next release... so I think if the purpose of this is retroactively modify the changelog in `1.0.0` we want to leave it out

Reply:
I don't think .changelog is needed since that was before this releases time. Yes Github release and also an update to our 1.0.x release notes would be much appreciated? https://developer.hashicorp.com/consul/docs/release-notes/consul-k8s/v1_0_x

Reply:
Ah, this went in early today for docs: https://github.com/hashicorp/consul/pull/18230/files (not quite backported yet), but it didn't touch the release notes page.

I'll open a PR for the release notes page.

Reply:
Release notes docs PR: https://github.com/hashicorp/consul/pull/18293